comms_webrtc.orogen

```diff
@@ -2,54 +2,48 @@ name "comms_webrtc"
 # Optionally declare the version number
 # version "0.1"
 
-# If new data types need to be defined, they have to be put in a separate C++
-# header, and this header will be loaded here
 import_types_from "comms_webrtcTypes.hpp"
-# Finally, it is pretty common that headers are directly loaded from an external
-# library. In this case, the library must be first used (the name is the
-# library's pkg-config name) and then the header can be used. Following Rock
-# conventions, a common use-case would be:
-#
-# using_library "comms_webrtc"
-# import_types_from "comms_webrtc/CustomType.hpp"
-
-# If this project uses data types that are defined in other oroGen projects,
-# these projects should be imported there as well.
-import_types_from "std"
-# import_types_from "base"
-
-# Declare a new task context (i.e., a component)
-#
-# The corresponding C++ class can be edited in tasks/Task.hpp and
-# tasks/Task.cpp, and will be put in the comms_webrtc namespace.
+import_types_from "iodrivers_base"
+import_types_from "base"
+
+using_library "libdatachannel"
+using_library "jsoncpp", typekit: true
```

Comment:
If I put `false`, it is not possible to include `"json/json.h"`in the `comms_webrtcTypes.hpp`. 
It says that can't find the header..

Reply:
That would be because MessageDecoder is there. And should not be.

Only put types there that are to be used on the component(s) interface(s)

You can put the rest in separate headers, for instance in e.g. `tasks/MessageDecoder.hpp`